testing/features/support/driver.rb

```diff
@@ -25,7 +25,7 @@ def setup_capybara
     Capybara.configure do |config|
       config.run_server = false
       config.default_driver = :selenium
-      config.default_max_wait_time = 0
+      config.default_max_wait_time = 1
```

Comment:
Not sure if we would want this to be 0?  I'm aware I've only used capybara selectors so not sure if there is a SitePrism thing I should be using?  Or if this should just be set for the nav tests (the delay is required to wait for the More and Close buttons to be present).

Reply:
So this is a setting we should be very careful with editing. Ideally we would use in-line waiters. Any declaration or invocation of a CNE (Capybara::Node::Element), either in-line with capybara or using SitePrism, can pass it.

i.e.

```
page.find("button", wait: 5) # Wait up to 5secs for button
@component.navigation(wait: 3) # Wait up to 3secs for navigation SitePrism element
element :navigation, ".js-cads-greedy-nav", wait: 3 # Wait up to 3 secs to "set" the SitePrism reference
```

If you find you need to set an implicit waiting strategy, and you're using something that isn't Mojave Safari, I'd be very intrigued why it's needed.

Reply:
Agreed to use inline waiting where required